test/clj/rems/api/test_licenses.clj

```diff
@@ -59,131 +59,180 @@
                          read-body)]
             (is (= id (:id data)))))))
 
-    (testing "create linked license"
-      (let [command {:licensetype "link"
-                     :organization "test-organization"
-                     :localizations {:en {:title "en title"
-                                          :textcontent "http://example.com/license/en"
-                                          :attachment-id nil}
-                                     :fi {:title "fi title"
-                                          :textcontent "http://example.com/license/fi"
-                                          :attachment-id nil}}}]
-        (testing "as owner"
-          (let [id (:id (create-license "owner" command))]
+    (testing "can't create license as organization owner with incorrect organization"
+      (is (response-is-forbidden? (api-response :post "/api/licenses/create"
+                                                {:licensetype "link"
+                                                 :organization "organization2"
+                                                 :localizations {:en {:title "en title"
+                                                                      :textcontent "http://example.com/license/en"}
+                                                                 :fi {:title "fi title"
+                                                                      :textcontent "http://example.com/license/fi"}}}
+                                                api-key "organization-owner1"))))
+
+    (doseq [user [org-owner owner]]
+      (testing user
+        (testing "create link license"
+          (let [command {:licensetype "link"
+                         :organization "organization1"
+                         :localizations {:en {:title "en title"
+                                              :textcontent "http://example.com/license/en"
+                                              :attachment-id nil}
+                                         :fi {:title "fi title"
+                                              :textcontent "http://example.com/license/fi"
+                                              :attachment-id nil}}}
+                id (:id (create-license user command))]
+              (is id)
+              (testing "and fetch"
+                (let [license (-> (request :get (str "/api/licenses/" id))
```

Comment:
Should these be using `api-call`? There's some uses so I guess could refactor some more.